src/components/Calculator.js

```diff
@@ -1,53 +1,74 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
 import React from 'react';
 import Buttons from './Buttons.js';
 import Input from './Input.js';
+import calculate from '../logic/calculate.js';
 
 class Calculator extends React.Component {
```

Comment:
- [ ]  Please import operate.js here just like calculate.js for the app to work fine.

Reply:
But Calculator is not using operate.js only calculate.js is using it. So we shouldn't import it here. But anyway if you want.